### PR TITLE
Add missing security options to /info response

### PR DESCRIPTION
**Pull request by SoMuchForSubtlety**
The docker API reports running rootless and SELinux as security options  in the `GET /info` response, so podman should do the same for the compat API.

I'm not sure how to test this. Should I extend the python REST test? It currently only checks for status code.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
Could you add a test to test/apiv2/?

File: pkg/api/handlers/compat/info.go [hunk 1]
```diff
@@ -181,6 +182,13 @@ func getSecOpts(sysInfo *sysinfo.SysInfo) []string {
 		// FIXME: get profile name...
 		secOpts = append(secOpts, fmt.Sprintf("name=seccomp,profile=%s", "default"))
 	}
+	if rootless.IsRootless() {
+		secOpts = append(secOpts, "name=rootless")
+	}
+	if selinux.GetEnabled() {
+		secOpts = append(secOpts, "name=selinux")
```
Comment (rhatdan) [inline pkg/api/handlers/compat/info.go:189]: These look good to me, although I don't know what the docker protocol expects?

Should we also return info for apparmor?

Comment (SoMuchForSubtlety) [reply pkg/api/handlers/compat/info.go:189]: Podman already reports apparmor. This[^1] is the equivalent code from moby, they also report `userns` and `cgroupns`.

[^1]: https://github.com/moby/moby/blob/5bf405b2afea6daf3960ad91583b0d290afd843f/daemon/info.go#L150-L175

Comment (rhatdan) [reply pkg/api/handlers/compat/info.go:189]: Awesome, LGTM

File: test/apiv2/python/rest_api/test_v2_0_0_system.py [hunk 2]
```diff
@@ -16,7 +16,9 @@ def test_info(self):
         r = requests.get(self.podman_url + "/v1.40/info")
         self.assertEqual(r.status_code, 200, r.text)
         self.assertIsNotNone(r.content)
-        _ = r.json()
+        response = r.json()
+        self.assertIn("name=rootless", response["SecurityOptions"])
```
Comment (rhatdan) [inline test/apiv2/python/rest_api/test_v2_0_0_system.py:20]: Should add a check here on whether the test is running as non-root.

File: test/apiv2/python/rest_api/test_v2_0_0_system.py [hunk 3]
```diff
@@ -16,7 +16,9 @@ def test_info(self):
...
+        self.assertIn("name=selinux", response["SecurityOptions"])
```
Comment (rhatdan) [inline test/apiv2/python/rest_api/test_v2_0_0_system.py:21]: This should be within a if SELinuxEnabled block.

File: test/apiv2/python/rest_api/test_v2_0_0_system.py (hunk 2 above)
Comment (SoMuchForSubtlety) [reply test/apiv2/python/rest_api/test_v2_0_0_system.py]: done

File: test/apiv2/python/rest_api/test_v2_0_0_system.py (hunk 3 above)
Comment (SoMuchForSubtlety) [reply test/apiv2/python/rest_api/test_v2_0_0_system.py]: done

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**